src/test/java/graphql/annotations/connection/GraphQLConnectionTest.java

```diff
@@ -396,4 +422,42 @@ public String getCursor(Obj entity) {
 
     }
 
+    /*
+     * double @GraphQLNonNull on @GraphQLList like [ObjectType!]!
+     */
+    public static class DoubleNonNullIterableTypeFunction implements TypeFunction {
```

Comment:
As I have experimented, this is the best way to implement `double @GraphQLNonNull like [ObjectType!]!`.
How about to make it builtin?

Reply:
interesting.. we have to think about it (I don't think we handle double graphql non-null at all, not only in connection)